Makefile.js

```diff
@@ -130,12 +134,22 @@ function release(type) {
     echo("Publishing to git");
     exec("git push origin master --tags");
 
-    echo("Publishing to npm");
-    exec("npm publish");
-
-    echo("Generating site");
-    target.gensite();
-    target.publishsite();
+    // now push the changelog...changes to the tag
+    echo("Publishing changes to github release");
+    // this requires a github API token in process.env.GITHUB_TOKEN
```

Comment:
You should check if this environment variable is set. If not, you should print out an error and exit or continue without pushing the changelog to the github release notes.